### ISSUE TITLE
Blender 2.93 and Addon 0.3 Python error

### ISSUE THREAD

**Issue by 3dcinetv** [Gorgious56/MazeGenerator#12]
Hello, I tried to run the maze addon on Blender 2.93.11 (latest 2.93 version as of Nov 2022) and the addon 0.3 error, when generating the maze says:
Python: Traceback (most recent call last):
  File "C:\Users\3DCompositor\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\Maze Generator\ui\panels\path.py", line 35, in draw
AttributeError: 'MGProperties' object has no attribute 'path'

location: <unknown location>:-1

**Comment by 3dcinetv** [comment]
I installed it on 2.92, version 0.3 - Whenever I "undo", Blender crashes.

**Comment by Gorgious56** [comment]
Yup, the addon is only supported up to 2.83 sorry ! I responded on your other issue thread
